datadog/tests/resource_datadog_dashboard_test.go

```diff
@@ -277,6 +277,24 @@ resource "datadog_dashboard" "ordered_dashboard" {
 				}
 				display_type = "bars"
 			}
+			request {
+                audit_query {
+                    index = "*"
+                    search_query = ""
+                    group_by {
+                        facet = "@metadata.api_key.id"
+                        sort_query {
+                            aggregation = "count"
+                            order = "desc"
+                        }
+                        limit = "10"
+                    }
+                    compute_query {
+                        aggregation = "count"
+                    }
+                }
+                display_type = "line"
```

Comment:
nit: can we format this accordingly? Looks like this section needs a couple of tabs.

Reply:
Fixed the formatting issues